pages/docs/tutorials/jvm-get-started.md

```diff
@@ -2,61 +2,81 @@
 type: tutorial
 layout: tutorial
 title:  "Getting Started with IntelliJ IDEA"
-description: "This tutorial walks you through creating a simple Hello World application using IntelliJ IDEA."
-authors: Hadi Hariri, Roman Belov
-date: 2020-01-23
+description: "This tutorial demonstrates how to use IntelliJ IDEA for creating a console application. "
+authors: Kate Volodko
+date: 2020-07-07
 showAuthorInfo: false
 ---
-## Setting up the environment
 
-In this tutorial we're going to use IntelliJ IDEA. To get started, install a recent version of IntelliJ IDEA.
+To get started, install a recent version of [IntelliJ IDEA](http://www.jetbrains.com/idea/download/index.html).
 
-Kotlin is bundled with IntelliJ IDEA starting from version 15.
+## Create an application 
 
-You can download the free [Community Edition][intellijdownload] (or full-fledged [Ultimate Edition][intellijdownload]) from the [JetBrains website][jetbrains].
+Once you've installed IntelliJ IDEA, it's time to create your first Kotlin application.
 
-As an alternative to using IntelliJ IDEA, you can compile and execute Kotlin applications using the command line compiler. For details, see [Working with the Command Line Compiler][getting_started_command_line].
+1. In IntelliJ IDEA, select **File** \| **New** \| **Project**.
+2. In the panel on the left, select **Kotlin**.
+3. Enter a project name, select **Console Application** as the project template, and click **Next**.
+   
+   ![Create a console application]({{ url_for('tutorial_img', filename='getting-started/jvm-new-project-1.png') }})
+   
+   By default, your project will use the build system Gradle with Kotlin DSL.
 
-If you are new to the JVM and Java, check out the [JVM Minimal Survival Guide](http://hadihariri.com/2013/12/29/jvm-minimal-survival-guide-for-the-dotnet-developer/).
-If you are new to IntelliJ IDEA, check out the [The IntelliJ IDEA Minimal Survival Guide](http://hadihariri.com/2014/01/06/intellij-idea-minimal-survival-guide/).
+3. Accept the default configuration on the next screen and click **Finish**.
+  
+   ![Configure a console application]({{ url_for('tutorial_img', filename='getting-started/jvm-new-project-2.png') }}) 
 
-## Creating a new project
-Once you have IntelliJ IDEA installed, it's time to create your first Kotlin application.
-1. Create a new __Project__ from __File \| New__. Select the __Kotlin \| JVM \| IDEA__ project type.
+   Your project opens. By default, you see the file `build.gradle.kts`, which is the build script created by the Project 
+   Wizard based on your configuration. It includes the `kotlin("jvm")` plugin and dependencies required for your console application.
 
-   ![Kotlin New Project]({{ url_for('tutorial_img', filename='getting-started/new_project_step1.png') }})
+3. Open the file `main.kt` in **src** \| **main** \| **kotlin**.  
```

Comment:
Please use conventional path separators to avoid confusion: `src/main/kotlin``
Vertical lines are for describing locations in UI (menus, drop-downs, etc.)